code/modules/mining/mine_turfs.dm

```diff
@@ -430,7 +431,7 @@ var/list/mining_floors = list()
 /turf/simulated/mineral/random
 	name = "Mineral deposit"
 	var/mineralSpawnChanceList = list("Uranium" = 5, "Platinum" = 5, "Iron" = 35, "Carbon" = 35, MATERIAL_DIAMOND = 1, MATERIAL_GOLD = 5, MATERIAL_SILVER = 5, MATERIAL_PHORON = 10)
-	var/mineralChance = 1.5 //means 1% chance of this plot changing to a mineral deposit
+	var/mineralChance = 25 //means 25% chance of this plot changing to a mineral deposit
```

Comment:
25% is a completely crazy mineral chance, have you tested this? need to see it in action

Reply:
It's not too bad, honestly. 25% is also lore appropriate. You know Aegis 7 was a literal goldmine right?